src/test/tests/maven.test.ts

```diff
@@ -320,31 +320,40 @@ describe('Maven Tests', async () => {
 
```

Comment:
This change looks like a mistake and probably does not belong to this pull request.
Same goes for mavenUpdate.test.ts.